src/test/java/net/minecraftforge/debug/misc/GameTestTest.java

```diff
@@ -0,0 +1,72 @@
+package net.minecraftforge.debug.misc;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.gametest.framework.GameTest;
+import net.minecraft.gametest.framework.GameTestGenerator;
+import net.minecraft.gametest.framework.GameTestHelper;
+import net.minecraft.gametest.framework.GameTestRegistry;
+import net.minecraft.gametest.framework.TestFunction;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.Rotation;
+import net.minecraftforge.fml.common.Mod;
+
+import java.util.List;
+
+@Mod(GameTestTest.MODID)
+public class GameTestTest {
+    public static final String MODID = "gametest_test";
+    public static final boolean ENABLE = true;
+
+    public GameTestTest() {
+        if (ENABLE) {
+            // Registers all methods annotated with @GameTest or @GameTestGenerator to the GameTestRegistry
+            GameTestRegistry.register(this.getClass());
+        }
+    }
+
+    // An example game test
```

Comment:
Is there a reason the comments on this method and on `generateTests` are formatted as regular comments rather than javadocs?